src/styles/popup.css

```diff
@@ -1,34 +1,65 @@
+
+/* variables
+_________________________________________________________*/
+:root {
```

Comment:
I would like to use css custom properties for all the color logic. Setting and getting them in a Chrome extension has proven a little more elusive than I expected. But this will at least help consolidate our style values.